### ISSUE TITLE
Pre and post script

### ISSUE THREAD

**Issue by Anuskuss** [Heroic-Games-Launcher/HeroicGamesLauncher#2776]
### Problem description

I need to run a DualShock wrapper before starting the game otherwise I won't have controller input.

### Feature description

Add the ability to define scripts that run before (and/or after) the game. Probably only needed for Windows games.

**Comment by arielj** [comment]
Closing as duplicate of https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/2507
